internal/awsutils/ec2_client.go

```diff
@@ -26,9 +28,11 @@ type Client struct {
 }
 
 // NewEc2Client creates a new AWS client in the given region.
-func NewEc2Client(region string) (*Client, error) {
+func NewEc2Client() (*Client, error) {
 	// Initialize client session configuration.
-	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithRegion(region))
+	vkcfg := vkconfig.Config()
+	httpClient := http.NewBuildableClient().WithTimeout(time.Second * time.Duration(vkcfg.AWSClientTimeoutSeconds))
+	cfg, err := config.LoadDefaultConfig(context.TODO(), config.WithHTTPClient(httpClient))
```

Comment:
Ideally we should set overall timeout for API but we can revisit this later. Can you add both the above links as comment.

Reply:
@SaranBalaji90 Shouldn't this go in issues or `BacklogFodder.md`?  I thought we were removing `TODO`s from the code going forward.

Reply:
I was asking to add the SDK doc links as comment so that its easier for someone to refer when needed.